src/server/cmd/pachd/main.go

```diff
@@ -86,7 +88,8 @@ func main() {
 	// set GOMAXPROCS to the container limit & log outcome to stdout
 	maxprocs.Set(maxprocs.Logger(log.Printf)) //nolint:errcheck
 	log.Infof("version info: %v", version.Version)
-	ctx := context.Background()
+	ctx, cancel := signal.NotifyContext(context.Background(), notifySignals...)
```

Comment:
is there a way we can get the log for the error that caused the shutdown?

Reply:
It gets logged in `cmdutil.Main`.